app/api-v1/services/orderService.js

```diff
@@ -0,0 +1,17 @@
+const { postOrderDb } = require('../../db')
+const { BadRequestError } = require('../../utils/errors')
+
+async function postOrder(reqBody) {
+  // Will add a get function at a later date to check for duplication
+
+  const result = await postOrderDb(reqBody)
+  if (!result) {
+    throw new BadRequestError({ message: '400 Bad Request', service: 'order' })
```

Comment:
`BadRequestError` already has `code: 400`, the `message` part of the error is for providing more detail i.e. what was bad about the request. Also is this a `400`, when do we expect `postOrderDb` to return a falsey value?